src/Layout.js

```diff
@@ -59,28 +59,6 @@ var computeLayout = (function() {
     'column-reverse': 'height'
   };
 
-  function capitalizeFirst(str) {
-    return str.charAt(0).toUpperCase() + str.slice(1);
```

Comment:
Yeah, this was stupidly slow. I was just lazy when I wrote the js version and since we never used it in prod (only the c and java), I never got around to make it faster. Thanks for doing that!